frontend/src/component/JobPostingGallery.tsx

```diff
@@ -8,9 +9,27 @@ export type JobPostingGalleryProps = {
 export default function JobPostingGallery(props: JobPostingGalleryProps) {
 
     return <div>
-        {props.user?.jobPostings.length !== 0 && props.user.jobPostings.map(
-            jobPosting => <JobPostingCard jobPosting={jobPosting} key={jobPosting.id}/>)
-        }
+
+        <Grid container spacing={2} sx={{color: 'white'}}>
+            <Grid item>
+                <Box>
+                    Interested in
+                    {props.user?.jobPostings.length !== 0 && props.user.jobPostings.filter(jobPosting => jobPosting.status === "INTERESTED_IN").map(
```

Comment:
props.user?.jobPostings.length !== 0 
might be props.user?.jobPostings.length > 0

Reply:
Habs jetzt geändert aber mochte meins lieber haha